src/Microsoft.DotNet.Wpf/src/PresentationCore/MS/internal/AppModel/CookieHandler.cs

```diff
@@ -14,22 +14,16 @@
 //
 //
 
-using System;
 using System.Net;
-using System.Security;
-using System.Diagnostics;
 using System.ComponentModel;
 using System.Runtime.InteropServices;
 using System.Diagnostics.CodeAnalysis;
-
-using System.Windows;
-using System.Windows.Interop;
 using MS.Win32;
 using MS.Internal.PresentationCore;
 
 namespace MS.Internal.AppModel
 {
-static class CookieHandler
+    static class CookieHandler
```

Comment:
Couple of files have this change, doesn't seem intentional

Reply:
They should be indented like that, the tooling did it. There is a meta issue of indenting the types properly in some cases.

Reply:
Yes but it doesn't indent the whole class, just this line? (+leading trivia)